db/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.0.2 on 2023-10-26 20:47
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
change ' to ", use Black

Reply:
do you want me to fix the double quotes? but this code is generated automatically by the command python manage.py makemigrations db
I don't understand the point of editing something

Reply:
it is good practice to use the same quotes for all of code lines